install_cli.php

```diff
@@ -515,6 +533,10 @@ class FakeLogger { public function __call($m, $a) { } }
 		echo "\tCleaning up demo data ... ";
 		/* Make sure not to delete the admin! */
 		if($module == 'Users'){
+			$GLOBALS['db']->query(
+				"DELETE FROM user_preferences WHERE id IN " .
+				"(SELECT md5(id) FROM $bean->table_name WHERE id != '1' AND id LIKE 'seed-%')"

```

Comment:
agree, actually you don't know what is `$bean->table_name`, it can contain invalid string...